core/cmd/remote_client.go

```diff
@@ -197,6 +201,71 @@ func (cli *Client) ChangePassword(c *clipkg.Context) (err error) {
 	return nil
 }
 
+// Profile will collect pprof metrics and store them in a folder.
+func (cli *Client) Profile(c *clipkg.Context) error {
+	seconds := c.Uint("seconds")
+	baseDir := c.String("output_dir")
+	if seconds >= uint(cli.Config.HTTPServerWriteTimeout().Seconds()) {
+		return cli.errorOut(errors.New("profile duration should be less than server write timeout."))
+	}
+	genDir := fmt.Sprintf("%sdebuginfo-%d", baseDir, time.Now().Unix())
```

Comment:
Maybe don't count on the user including a trailing `/` here?
```suggestion
	genDir := filepath.Join(baseDir, fmt.Sprintf("debuginfo-%d", time.Now().Unix()))
```
Also could consider `time.RFC3339` instead of unix for human readability?